client/my-sites/upgrades/checkout/credit-card-payment-box.jsx

```diff
@@ -38,7 +38,9 @@ var CreditCardPaymentBox = React.createClass( {
 					initialCard={ this.props.initialCard }
 					transaction={ this.props.transaction } />
 
-				<TermsOfService />
+				<TermsOfService
+					hasRenewableSubscription={ cartValues.cartItems.hasRenewableSubscription( cart ) }
+					/>
```

Comment:
I don't think there is a rule for that, but in most cases closing tag lands after last property.